ConcurrentPercolator/ConcurrentPercolator.tla

```diff
@@ -84,16 +83,30 @@ hasStaleLock(k, ts) ==
 
 \* Returns the writes with start_ts equals to $ts$.
 findWriteWithStartTS(k, ts) ==
-  {w \in Range(key_write[k]) : (w.type = "write" /\ w.start_ts = ts)}
-
+  {w \in Range(key_write[k]) : (w.type = "write" /\ w.start_ts = ts)}      
+   
 \* Returns the writes with commit_ts equals to $ts$.
 findWriteWithCommitTS(k, ts) ==
   {w \in Range(key_write[k]) : (w.type = "write" /\ w.ts = ts)}
 
 \* Returns TRUE if there is a rollback for key $k$ at timestamp $ts$.
```

Comment:
\* Returns TRUE if there is a rollback for key $k$ that is not deleted logically at timestamp $ts $.